README.md

```diff
@@ -2,11 +2,8 @@
 
 datadog-pgo is a tool for integrating continuous [profile-guided optimization](https://go.dev/doc/pgo) (PGO) into your Go build process. It fetches representative CPU profiles from Datadog and merges them into a `default.pgo` file that is used by the Go toolchain to optimize your application.
 
-## ⚠️ Public Beta
+You can learn more about this feature in our [official documentation](https://github.com/DataDog/documentation/blob/master/content/en/profiler/guide/save-cpu-in-production-with-go-pgo.md) as well as in [our announcement blog post](https://docs.datadoghq.com/profiler/guide/save-cpu-in-production-with-go-pgo/).
```

Comment:
For the official documentation link, perhaps you want this instead of a link to the markdown on github? https://docs.datadoghq.com/profiler/enabling/go/#save-up-to-14-cpu-in-production-with-pgo

Reply:
Ugh, both links were wrong 🤦 . Thanks for catching this. Should be fixed now. PTAL again.